ext/websocket/02_websocketstream.js

```diff
@@ -370,7 +395,14 @@
         this[_earlyClose] = true;
       } else if (this[_closed].state === "pending") {
         PromisePrototypeCatch(
-          core.opAsync("op_ws_close", this[_rid], code, closeInfo.reason),
+          PromisePrototypeThen(
+            core.opAsync("op_ws_close", this[_rid], code, closeInfo.reason),
+            () => {
+              setTimeout(() => {
+                this[_closeSent].resolve(new Date().getTime());
+              }, 0);
+            },
+          ),
```

Comment:
nitpick: dont use `PromisePrototypeCatch` & `PromisePrototypeThen`, but just `PromisePrototypeCatch` and 3rd param

Reply:
✅ done